runelite-client/src/main/java/net/runelite/client/plugins/coxhelper/CoxInfoBox.java

```diff
@@ -75,14 +75,19 @@ public Dimension render(Graphics2D graphics)
 		{
 			prayAgainstPanel.getChildren().clear();
 
+			if (!plugin.isConfigPrayAgainstOlm())
+			{
+				return null;
+			}
+
 			final PrayAgainst prayAgainst = plugin.getPrayAgainstOlm();
 
-			if (plugin.getPrayAgainstOlm() == null && !plugin.isConfigPrayAgainstOlm())
+			if (prayAgainst == null)
```

Comment:
i would rather you just add `|| !plugin.isConfigPrayAgainstOlm()` rather than having 2 separate IF statements to achieve the same result.

Reply:
The way he did it was more appropriate, first checking if its actually enabled, then initializing the prayAgainst, then null checking it.

Reply:
Reverted